includes/definitions/xerox.yaml

```diff
@@ -10,3 +10,4 @@ discovery:
         - Xerox Phaser
         - Xerox WorkCentre
         - FUJI XEROX DocuPrint
+    - sysDescr_regex: '/^.*DocuPrint.*/'
```

Comment:
This is actually the same as `- sysDescr: DocuPrint`  the sysDescr section searches for the given string anywhere in the sysDescr